tutorials/week1-installing-postman.md

```diff
@@ -0,0 +1,186 @@
+Once Postman is installed:
+1. Open the Postman application.
+2. You will be prompted to sign in or create a Postman account.
+
+## Using Postman: Basic Steps
+
+### Step 0: Ensure your backend is running
+
+
+### Step 1: Launch Postman
+After installation, launch Postman. You'll see a workspace where you can start creating requests.
+
+### Step 2: Make Your First API Request
+1. Click on the "New" button at the top-left and select **HTTP**.
+
+![Postman1](./assets/week1-installing-postman/Postman1.png)
+
+2. In the request window, choose the type of request method (GET, POST, PUT, DELETE) from the dropdown menu (default is GET).
+3. Enter the URL of the API you want to test in the input field (e.g., `https://localhost:8000/api/users`).
+
+![Postman2](./assets/week1-installing-postman/Postman2.png)
+
+4. To send a POST request you need to include the Body as well.
+    - Go to the **Body** tab.
+    - Select **raw**.
+    - Choose **JSON** as the format.
+    - Enter the JSON data in this format:
+    ```json
+    {
+        "username": "testuser",
+        "email": "test@example.com"
+    }
+    ```
+![Postman3](./assets/week1-installing-postman/Postman3.png)
+
+5. Click the **Send** button.
+
+### Step 3: View the Response
+- After clicking "Send," Postman will display the API response below the request window.
+- You will see the **Status Code**, **Response Body**, **Headers**, and **Cookies** (if applicable).
+
+## Saving API Requests and Using History
+
+One of Postman's key features is the ability to **save API requests** for future use. This can save you time when testing or working on multiple endpoints. Additionally, Postman keeps track of your request history, allowing you to quickly revisit previous requests without re-entering all the details.
+
+To learn more about using Postman's history, visit: [Postman History](https://learning.postman.com/docs/getting-started/basics/navigating-postman/#history).
+
+## Organizing Requests into Collections
+
+Postman also allows you to save and organize requests into **collections**. A collection is essentially a folder where you can store multiple requests, making it easier to manage large projects and share API workflows with your team.
+
+For more information on creating and using collections, refer to the Postman documentation: [Postman Collections](https://learning.postman.com/docs/sending-requests/intro-to-collections/).
+
+These features enhance your ability to efficiently manage, organize, and track your API development process within Postman.
+
+
+## Recommended Tools
+
+In addition to using the Postman desktop app, you can also integrate Postman with Visual Studio Code through the Postman extension. This extension allows you to send API requests directly from your code editor. 
+
+You can find the extension and its documentation here: [Postman VS Code Extension](https://learning.postman.com/docs/getting-started/basics/about-vs-code-extension/).
```

Comment:
Should we add a link to the [Postman Getting Started](https://learning.postman.com/docs/getting-started/first-steps/sending-the-first-request/) guide here under a "Useful Resources" section? It also contains an example that will run independent of the assignments (uses `postman-echo.com/get`) in case students want to follow it.

Reply:
This tutorial is quite elaborate already. I feel this would just add more content, and not much value.